docs/controls/MarkdownTextBlock.md

```diff
@@ -0,0 +1,41 @@
+# MarkdownTextBlock XAML Control 
+
+The *MarkdownTextBlock XAML control* provides full markdown parsing and rendering for Universal Windows Apps. Originally created for the open source reddit app Baconit, the control was built to be simple to use and very efficient. One of the main design considerations for the control was it needed to be performant enough to provide a great user experience in virtualized lists. With the custom markdown parser and efficient XAML rendering, we were able to achieve excellent performance; providing a smooth UI experience even with complex Markdown on low end hardware.
+
+Under the hood, the control uses XAML sub elements to build the visual rendering tree for the Markdown input. We chose to use full XAML elements over using the RichEditTextBlock control because the RichEditTextBlock isn’t flexible enough to correctly render all of the standard Markdown styles.
+
+## Syntax
+
+```xaml
+
+ <controls:MarkdownTextBlock
+    Text="**This is *Markdown*!"
+    OnMarkdownRendered="MarkdownText_OnMarkdownRendered"
+    OnLinkClicked="MarkdownText_OnLinkClicked"
+    Margin="6">
+</controls:MarkdownTextBlock>
+
+```
+
```

Comment:
Could it be possible to show all possible examples with Markdown elements ? I mean examples for Headers, List and more important some elements that are not so often used.